src/components/DropdownMenuWc/DropdownMenuWc.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DropdownMenuItems from '../DropdownMenuItems/DropdownMenuItems';
+
+/**
+ * @uxpincomponent
+ */
+export default function DropdownMenuWc(props) {
+  const { active, mode, visibleItems } = props;
+
+  // #region Methods
+  const getDropdownItems = () => {
+    const items = [];
+
+    for (let i = 1; i <= 10; i++) {
+      const title = props[`item${i}`];
+
+      if (title) {
+        items.push({
+          title,
+          iconLeft: props[`iconLeft${i}`],
+          iconRight: props[`iconRight${i}`],
+          description: props[`item${i}Description`],
+          onSelect: props[`select${i}`],
+        });
+      }
+    }
+    return items;
+  };
+  // #endregion
+
+  // #region Attributes
+  const items = getDropdownItems(props);
+
+  const hasDescription = items.some((item) => item.description);
+  // #endregion
+
+  // TODO: Implement custom width and height for dropdown menu (both with items and when empty) on chi
+  // TODO: Retain selection with active selected items should be implemented on chi
+  // TODO: Visible items should be able to add scroll on dropdown menu and must be compatible with custom height
+  // TODO: Appropriate padding must be provided between item and icon right
+  return (
+    <chi-dropdown
+      active={active}
```

Comment:
```suggestion
      active
```

Reply:
@josecarlosgonzalezv We need this active flag to toggle dropdown menu to show and hide to be in sync with current implementation.